Source/pack.cpp

```diff
@@ -556,6 +556,26 @@ bool UnPackNetPlayer(const PlayerNetPack &packed, Player &player)
 	for (int i = 0; i < NUM_INVLOC; i++) {
 		if (!UnPackNetItem(player, packed.InvBody[i], player.InvBody[i]))
 			return false;
+		auto loc = player.GetItemLocation(player.InvBody[i]);

```

Comment:
Because `loc` might be `ILOC_NONE` in the case where there is no item in the slot.
```suggestion
		if (item.isEmpty())
			continue;
		auto loc = player.GetItemLocation(player.InvBody[i]);
```